Storage/Storage/Model/DashboardCard.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+/// Represents a card on the Dashboard screen.
+public struct DashboardCard: Codable, Hashable, Equatable {
+    /// The type of dashboard card.
+    public let type: CardType
+
+    /// Whether the card is enabled in the Analytics Hub.
+    public var enabled: Bool
+
+    public init(type: CardType, enabled: Bool) {
+        self.type = type
+        self.enabled = enabled
+    }
+
+    /// Types of cards to display on the Dashboard screen.
+    /// The order of the cases in this enum defines the default order.
```

Comment:
Nice idea. 👏 

I hope in future, we plan to use a for loop (using the `CaseIterable` confirmation) to render the cards in https://github.com/woocommerce/woocommerce-ios/pull/12449/files#diff-5e4b4001de415046921cf84e4ce2b6c16646ae1bb2067aa6d9ec1fbfa2a855e5R324-R326. 

This way, changing the order in this `enum` changes the default order automatically.

Reply:
Yup the logic in `DashboardViewModel` is to be updated when we update the persistence for the dashboard cards.